cmd/utils.go

```diff
@@ -686,6 +686,13 @@ func CopyDir(log *LoggingConfig, fromDir string, toDir string) error {
 // CheckPrereqs checks the prerequisites to run the CLI
 func CheckPrereqs(config *RootCommandConfig) error {
 	if config.Buildah {
+		buildahCmd := "buildah"
+		buildahArgs := []string{"containers"}
+		checkBuildahCmd := exec.Command(buildahCmd, buildahArgs...)
+		_, buildahCmdErr := checkBuildahCmd.Output()
+		if buildahCmdErr != nil {
+			return errors.New("buildah does not seem to be installed or running - failed to execute buildah containers")
```

Comment:
I would suggest changing the message slightly and returning the error too. The message could be something like:
```
errors.Errorf("buildah does not seem to be installed or capable of running in this environment - failed to execute buildah containers: %v", buildahCmdErr)
```
`buildah containers` is a good test - but it may fail when the docker container that runs buildah isn't running on the correct platform (for example, macOS). In that case, you get an error even if buildah is installed correctly.